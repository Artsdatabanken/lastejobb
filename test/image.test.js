const { image } = require("..");

test("detect png", () => {
  const actual = image.getFileType("./test/image/test.png");
  expect(actual).toMatchSnapshot();
});

test("detect jpg", () => {
  const actual = image.getFileType("./test/image/test.jpg");
  expect(actual).toMatchSnapshot();
});
