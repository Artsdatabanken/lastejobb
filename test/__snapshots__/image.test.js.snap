// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`detect jpg 1`] = `"jpg"`;

exports[`detect png 1`] = `"png"`;
